proxy/src/coco/peer.rs

```diff
@@ -113,6 +125,67 @@ pub fn list_projects(peer: &PeerApi) -> Result<Vec<Project>, error::Error> {
         .collect()
 }
 
+/// Get all peer IDs and their branches.
+///
+/// # Errors
+///
+///   * [`error::Error::LibradLock`]
+///   * [`error::Error::Git`]
+pub fn remotes(
+    peer: &PeerApi,
+    owner: &User,
+    project_urn: &RadUrn,
+) -> Result<Vec<Remote>, error::Error> {
```

Comment:
`Vec` could be `NonEmpty` guarenteeing us to have our own `Remote`.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/636/commits/1cbb81935097a53360929384f82983c6caa956f4